src/QuadForm/Herm/Genus.jl

```diff
@@ -1452,17 +1452,11 @@ function representative(G::GenusHerm)
   P = _non_norm_primes(G.LGS)
   E = base_field(G)
   V = hermitian_space(E, _hermitian_form_with_invariants(base_field(G), rank(G), P, G.signatures))
-  @vprint :Lattice 1 "Finding maximal integral lattice\n"
```

Comment:
Ah yes, I didn't how those things work and while trying to read through the function I took them off because it seemed to be really heavy in the code and not convenient. I forgot to put them back in my last commit...